### DISCUSSION TITLE
Post your screenshots

### DISCUSSION THREAD

**Discussion by kimkulling** [assimp/assimp#4082] [Show and tell]
You can use this post to show your own screenshots!

Happy sharing,
Kim

**Comment by kimkulling** [comment]
I will start. My new Assimp-Viewer:
![205777881_10221240838303700_5679506273557061918_n](https://user-images.githubusercontent.com/2323156/133927415-1d3c8bc0-0e3a-4d52-8660-98450805d8c0.jpg)

**Comment by astrofra** [comment]
Let's shoot! Here is a realtime visualization of the [Poppy Ergo Junior](https://www.poppy-project.org/en/robots/poppy-ergo-jr/) robot, in the [HARFANG 3D engine](https://github.com/harfang3d/harfang3d). The 3D model is based on the [STL files](https://github.com/poppy-project/poppy-ergo-jr/tree/master/hardware/STL), imported using [ASSIMP](https://github.com/assimp/assimp) :

![unknown-14](https://user-images.githubusercontent.com/6149095/137546073-fb3b427e-3d3c-436d-ba48-0462bd3ff4b1.png)

![image0](https://user-images.githubusercontent.com/6149095/137546101-22bb9efe-6e18-4e46-a864-4f270a8f926d.jpg)

The STL files were [converted](https://github.com/harfang3d/harfang3d/tree/main/tools/assimp_converter) in our own HARFANG format, using ASSIMP, and the whole simulation is written/animated in [Python](https://www.python.org/) (code by https://github.com/scorpheus).

**Comment by jankrassnigg** [comment]
Privately I am using assimp for [ezEngine](http://ezengine.net) ([GitHub](https://github.com/ezEngine/ezEngine)). Both for static mesh import and animation import. It is much nicer to work with assimp than using the FBX SDK (which I tried at some point). Overall assimp is really awesome, my main gripe has been that new versions often fixed some issues, but regressed in other cases (pretty much always related to animation data from FBX files).

![ez-assimp](https://user-images.githubusercontent.com/6001174/147709763-6fabf7cf-9a34-4391-a1db-6768190cc370.jpg)

Also, at work we happily use assimp to import gigantic models:
[Azure Remote Rendering](https://docs.microsoft.com/en-us/azure/remote-rendering/overview/about)

**Comment by ongamex** [comment]
![image](https://user-images.githubusercontent.com/6237727/148663487-0d872f7d-cd49-421d-857d-bf0222e5c41f.png)
Here is a screenshot for my engine (still work-in-progress https://github.com/ongamex/SGEEngine).

Assimp workes great with GLTF files so far, I love the embedded texture support, as not in blender I can just export the gltf file without dealing with manually exporting the in-blender painted textures!

**Reply by kimkulling** [reply]
Looks amazing!

**Comment by JvanderKaaij** [comment]
![image](https://user-images.githubusercontent.com/109142874/231459846-6b753886-5aa9-446d-bae9-e9134ca2f0b4.png)

Loading a self-modeled Helmet FBX into my custom OpenGL renderer. Assimp made this a lot easier to do!

**Comment by vhollander** [comment]
![image](https://user-images.githubusercontent.com/108892429/236723836-784e6edf-0561-440d-ac02-ab361775ca5f.png)

**Reply by kimkulling** [reply]
Nice, thanks for sharing!

**Comment by kimkulling** [comment]
The current status of the new Asset-Importer-Viewer is based on my renderer. I know that my renderer sucks. 
![assimp_view_status_ 2024-02-27 210229](https://github.com/assimp/assimp/assets/2323156/f172a2b9-de1d-4663-8161-65993bde532c)

**Comment by Meakk** [comment]
Reading a FBX file with [F3D](https://github.com/f3d-app/f3d)

![zelda](https://github.com/assimp/assimp/assets/6069752/712fb5e5-2757-48d8-8b4d-977a1084eee0)

Credit: [theStoff](https://sketchfab.com/theStoff)

**Comment by wbyates777** [comment]
330 Mb Enterprise mesh and 198 Mb dry dock mesh  loaded with assimp.

![Enterprise](https://github.com/user-attachments/assets/e7796ec0-beb5-472d-8f68-cfe70bd94fc8)

**Reply by kimkulling** [reply]
Love it!
